calculator-app/src/App.js

```diff
@@ -0,0 +1,12 @@
+import "./App.css";
+import Calculator from "./Calculator";
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
```

Comment:
Next time I would put start my app in this file to avoid having this middle man file